openfl/interface/pki.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""PKI CLI."""
+
+import logging
+import os
+from pathlib import Path
+
+from click import group
+from click import option
+from click import pass_context
+from click import Path as ClickPath
+
+from openfl.component.ca.ca import certify
+from openfl.component.ca.ca import get_bin_names
+from openfl.component.ca.ca import get_token
+from openfl.component.ca.ca import install
+from openfl.component.ca.ca import remove_ca
+from openfl.component.ca.ca import run_ca
+
+
+logger = logging.getLogger(__name__)
+
+CA_URL = 'localhost:9123'
+
+
+@group()
+@pass_context
+def pki(context):
+    """Manage Step-ca PKI."""
+    context.obj['group'] = 'pki'
+
+
+@pki.command(name='run')
+@option('-p', '--ca-path', required=True,
+        help='The ca path', type=ClickPath())
+def run(ca_path):
+    """Run CA server."""
+    ca_path = Path(ca_path)
+    step_config_dir = ca_path / 'step_config'
+    pki_dir = ca_path / 'cert'
+    pass_file = pki_dir / 'pass_file'
+    ca_json = step_config_dir / 'config' / 'ca.json'
```

Comment:
`step_config`, `cert`, `pass_file`, `config` use in many places, maybe make sense create variables for them

Reply:
Should you please create an issue